test/org/daisy/dotify/formatter/impl/page/FieldResolverTest.java

```diff
@@ -77,4 +81,23 @@ public void testGetWidth_01() throws TranslatorConfigurationException {
         assertEquals(10, resolver.getWidth(1, 8));
     }
 
+    @Test
+    public void testFieldWithSoftHyphen() {
+        new FieldResolver(
+            null,
+            new FormatterContext(
+                BrailleTranslatorFactoryMaker.newInstance(),
+                null,
+                new FormatterConfiguration.Builder("sv-SE", "bypass").build()),
+            null,
+            null
+        ).resolveField(
+            null,
+            new FieldList.Builder(Arrays.asList(new StringField("xxx\u00adxxx"))).build(),
+            new SimpleBrailleTranslator(
+                new DefaultBrailleFilter(new IdentityFilter(), "und", null, null),
+                new DefaultBrailleFinalizer(), "bypass"),
+            null
+        );
```

Comment:
What do you test here?
I see no `assert` statement

Reply:
Oh, right. I'm testing that the code doesn't raise an exception. But I'll add an expected result.

Reply:
>Oh, right. I'm testing that the code doesn't raise an exception. But I'll add an expected result.
Maybe add a comment that you test that no exception is raised?
Or maybe you can add `@Test(expected = None.class)` as in https://stackoverflow.com/a/44801486/7479415

Reply:
I'll just add an assertEquals on the return value.

Reply:
I've done this.